SocketRocket/Internal/Utilities/SRRandom.m

```diff
@@ -0,0 +1,26 @@
+//
+// Copyright (c) 2016-present, Facebook, Inc.
+// All rights reserved.
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree. An additional grant
+// of patent rights can be found in the PATENTS file in the same directory.
+//
+
+#import "SRRandom.h"
+
+#import <Security/SecRandom.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+NSData *SRRandomData(NSUInteger length)
+{
+    NSMutableData *data = [NSMutableData dataWithLength:length];
+    int result = SecRandomCopyBytes(kSecRandomDefault, data.length, data.mutableBytes);
+    if (result != 0) {
+        //TODO: (nlutsenko) Check if there was an error.
```

Comment:
This is one of those cases where I would actually think throwing an exception is the right thing to do. If this fails - it means that our entire security stack is likely to be busted.